### PR TITLE
Feature/47 services

### PR DESCRIPTION
**Pull request by msayoung**
Initial docs for the services feature. 
Mostly borrowed from Croydon. 
Is this a good structure? 

@willguv @finnlewis - any thoughts?

### CODE REVIEW

File: docs/src/config/services-feature.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+They automatically pull in the title and the summary of the page they're linking to.
+
+***Example:*** 
+
+*Examples include:*
+
+- *marriages and civil partnerships*
+- *travel support*
+- *births*
+
```
Comment (ekes) [inline docs/src/config/services-feature.md:85]: Assuming the topics functionality isn't pulled out this certainly will need explaining.

Something like.

Sub-landing pages can be grouped by topic. The list of links will have subheadings. If you want to use topics that are used in other services you can add this, the subheaded section will include a more link to another page with a list of all content in those topics from all services.

**Comment by msayoung** [comment]
I've done a second pass at some documentation for Services feature. 
Who'd like to review? 
@danchamp @willguv @finnlewis @jpoesen ?

https://deploy-preview-48--trusting-noyce-aebebc.netlify.app/features/services/ 
https://deploy-preview-48--trusting-noyce-aebebc.netlify.app/features/services/content-design.html
https://deploy-preview-48--trusting-noyce-aebebc.netlify.app/features/services/technical.html
https://deploy-preview-48--trusting-noyce-aebebc.netlify.app/features/services/background.html

On that last one (Background), do we have any background wireframes, spec or research that we could link to @willguv ?

**Comment by willguv** [comment]
@msayoung 

This is great, a few very minor things:

- On the top level page, we call things "Landing Page, Page" etc but then call them "Service Landing Page, Service Page" etc elswehere. I think we need to be consistent and use prefix everything with "Service"

Screenshot 2021-07-02 at 11 58 30

- Are the screenshots of sites with white backgrounds OK? I don't like borders around images, but feels like these need them for legibility

-  We need to mention something about the ability to target specific service status messages to the overall status page eg https://www.croydon.gov.uk/service-status in this section

- I skimmed Technical as don't really get it

If these things are fixed, think this is good to go. Shall we tackle Background separately?

- landing page and page patterns were built in Brighton very early on
- they were heavily influenced by GOV.UK patterns at the time. I can contact them and ask if they have any research
- Clearleft did the initial design work, which was modified by Dave:

http://design.brighton-hove.gov.uk/design-examples.php?p=service-hub
http://design.brighton-hove.gov.uk/design-examples.php?p=service-sub-hub
http://design.brighton-hove.gov.uk/design-examples.php?p=service-topic

- Maybe include these as a starter for 10?

**Comment by msayoung** [comment]
@willguv - thanks for the feedback. 
I've added a box-shadow to the screenshots, a piece about the service-status page, and a bit of the history. 
Can we merge? 
https://deploy-preview-48--trusting-noyce-aebebc.netlify.app/

**Pull request merged**